src/main/webapp/WEB-INF/component/activity-helper.jsp

```diff
@@ -84,10 +89,20 @@
      * @return String with the creation time layout section
      */
     public String formatCreationTime(Instant time) {
+/*
+
         // Todo: Change this to show in local time.
-        LocalDateTime datetime = LocalDateTime.ofInstant(time, ZoneOffset.UTC);
-        String timeStamp = DateTimeFormatter.ofPattern("yyyy-MM-dd hh:mm:ss").format(datetime);
-        return String.format("<strong>%s GMT: </strong>", timeStamp.toString());
+        ZoneId zoneId = ZoneId.of("America/Los_Angeles");
```

Comment:
@GustavoMeza you might have done this before. Do you want to help out? Thanks!

Reply:
I would be happy to help, but I don't understand how. 😄 
Isn't this ok?

Reply:
The code is updated so this is no longer valid :)